frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -361,8 +360,16 @@ export default class LineAreaBarChart extends Component {
       onHoverChange,
       onAddSeries,
       onRemoveSeries,
+      settings,
     } = this.props;
 
+    const orderedSeries =
+      (settings["graph.dimensions"]?.length > 1 &&
+        settings["series_order"]
+          ?.filter(_series => _series.enabled)
+          .map(_series => series[_series.originalIndex])) ||
```

Comment:
since you've named this value 
https://github.com/metabase/metabase/blob/1c321aa4969224e94e74944892468ceb2421a403/frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx#L27

what do you think about making this consistent? This could would also help differentiate the shape between `_series` and `series` in this context since it wasn't apparent.

```suggestion
          .map(orderedItem => series[orderedItem.originalIndex])) ||
```